edge/stack/stack.go

```diff
@@ -55,27 +55,27 @@ const (
 type engineType int
 
 const (
+	// TODO: consider defining this in agent.go or re-use/enhance some of the existing constants
```

Comment:
might as well make that decision before merge. either make the change or not, but don't see a reason to leave the comment in codebase

Reply:
I'm more inclined to leave this definitions in this package if they are intended for stack operations only, instead of moving them to `agent.go` where more global definitions should be. That is, I would simply delete the comment :)

Reply:
I actually left that TODO here purposely for review but since I need to merge a lot more PRs after that one, I'm afraid that doing too many refactors in that PR will lead to a lot of conflicts.

Instead, I've opened https://github.com/portainer/agent/issues/248 to track these pieces and I will work on them after I've finished merging most of the PRs that are waiting for rebase.